src/components/Token/Hero/token-hero-rings.json

```
@@ -0,0 +1,110 @@
+[
+    {
+      "id": "Ring1",
+      "duration": 20,
+      "direction": "normal",
+      "diameter":300,
```

You should run this file through a formatter. There are some formatting inconsistencies, e.g.

```suggestion
      "diameter": 300,
```

---

True. Probably Malay didn't have the formatter running on save actions.